Makefile

```diff
@@ -45,6 +45,14 @@ lint:
 	cargo $(CARGO_FLAGS) clippy --workspace --all-targets --all-features --no-deps -- -D warnings
 	pnpm --recursive run lint
 
+fix:
+	cargo fmt --all
+	cargo clippy --workspace --all-targets --all-features --no-deps --fix -- -D warnings
+
+fix-uncommitted:
+	cargo fmt --all
+	cargo clippy --workspace --all-targets --all-features --no-deps --fix --allow-dirty -- -D warnings
```

Comment:
With a bit of `Makefile`-fu one could probably avoid the duplication here

Reply:
I don't really have strong opinion on this, downside of current version is that it is a little bit harder to eye-parse it and understand what command we actually run, but it's not so hard so I'm ok with this downside. Downside of explicit version is probability to have different commands in `fix` and `lint` but I'm ok with this as well since it won't harm our code quality if we review CI changes properly and I believe we are strict enough in reviewing them

Reply:
No strong opinion either ways, but since we will have 3 targets using small variations I would be more inclined to go with the current (deduped) one.

Reply:
@aldur do you think we can resolve this thread?